zerver/lib/upload.py

```diff
@@ -139,12 +143,14 @@ def upload_image_to_s3(
 
 def get_file_info(request, user_file):
     # type: (HttpRequest, File) -> Tuple[str, str]
-    uploaded_file_name = user_file.name
+    uploaded_file_name = user_file.name.encode('ascii')
```

Comment:
It's because `user_file.name` is supposed to be in 'ascii' format but we get a unicode string at this place, I think Django assumes that all strings are in unicode format and converts them.